File: src/java/dto/Book.java
package dto;

import java.util.Date;

public class Book {

	public String Name;
	private String authorName;
	private String publisher;
	private int edition;
	private int quantity;
	private boolean isExchanged;
	private Date year;
	private String language;
	
	public String getName() {
		return Name;
	}
	public String getAuthorName() {
		return authorName;
	}
	public String getPublisher() {
		return publisher;
	}
	public int getEdition() {
		return edition;
	}
	public int getQuantity() {
		return quantity;
	}
	public void setQuantity(int quantity) {
		this.quantity = quantity;
	}
	public boolean isExchanged() {
		return isExchanged;
	}
	public void setExchanged(boolean isExchanged) {
		this.isExchanged = isExchanged;
	}
	public Date getYear() {
		return year;
	}
	public String getLanguage() {
		return language;
	}
	
	
	
}

File: data-tier/Models/Book.cs

namespace dal.Models;
public class Book
{
    public int Id { get; set; }
    public string Name { get; set; }
    public int Edition { get; set; }
    public string Publisher { get; set; }
    public string Author { get; set; }
    public int Quantity { get; set; }
    public Boolean IsExchanged { get; set; }
    public int Year { get; set; }
    public string Language { get; set; }

}